src/common/options.cc

```diff
@@ -5591,6 +5591,9 @@ std::vector<Option> get_global_options() {
     .set_flag(Option::FLAG_STARTUP)
     .set_description("The number of threads for serving alienized ObjectStore"),
 
+    Option("crimson_alien_thread_cpu_cores", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_description("Cpu cores on which alienstore threads will run"),
+
```

Comment:
I think maybe I should make "crimson_alien_thread_cpu_cores" mandatory, so that other people would know alienstore needs to run on different core. How do you like it?

Reply:
Yeah, this is also on the table. Although it complicates the deployment, it makes the problem explicit.

Reply:
probably we can make osd "smarted" when it comes to the CPU core assignment. but since a simple approach is to assign fixed set of CPU to reactor and alien threads, and we don't have a solution to migrate threads across cores. maybe we can defer this to a follow-up change? i think, this problem is very likely closely connected with cephadm-crimson integration work. we started to track used ports of a host in cephadm. in future, i think we might also want to track the used CPU core, nic, and the designated memory in cephadm. and let cephadm to update the related setting. so by then, we don't need to worry about `crimson_alien_thread_cpu_cores` anymore. 

to explain why crimson + alien is slow the troublesome. actually the same applies to classic + bluestore. what i can think of is to have a tool. something like "osd-doctor" which put a typical work load to local OSDs, and check the CPU usage on local host, and compare it with the `crimson_alien_thread_cpu_cores` to see if the settings make sense or not. 

but i don't think we need to fix this on day 1.